rllab/misc/logger.py

```diff
@@ -154,16 +153,17 @@ def log(s, with_prefix=True, with_timestamp=True, color=None):
 
 
 def record_tabular(key, val):
-    _tensorboard.record_scale(str(key), val)
+    _tensorboard_summary.record_scale(str(key), val)
```

Comment:
Why does it not show that I remove them?
And I found some features are the same in our codes. I think you could merge my code to yours and remove mine.

Reply:
> Why does it not show that I remove them?

It shows here: https://github.com/ryanjulian/rllab/pull/58/commits/fa6ac5ec47e2f51ede79b434f1bf30bea53fc9ca
But it shouldn't be removed.

> And I found some features are the same in our codes. I think you could merge my code to yours and remove mine.

Yeah, this could work. I will approve this and merge your code.